builder/repo_control.go

```diff
@@ -130,6 +131,17 @@ func (b *Builder) NewDNFConfIfNeeded() error {
 			return errors.Wrapf(err, "Failed to write to dnf file: %s", b.Config.Builder.DNFConf)
 		}
 	}
+
+	if b.Config.Mixer.LocalRepoDir != "" {
+		localRepo := path.Join(b.Config.Mixer.LocalRepoDir, "repodata")
```

Comment:
This is fragile, but I didn't want to waste time re-creating a the yum repo for each build. Open to suggestions here.

On a side note, I noticed that `mixer build all` does re-create the yum repo each time.